style/checkbox.less

```
@@ -10,6 +10,15 @@
   left: 0;
 }
 
+.rdg-checkbox-label-disabled {
+  cursor: default;
+
+  .rdg-checkbox {
+    border: 2px solid #ccc;
```

```suggestion
    border-color: #ccc;
```
That should be enough.